.github/workflows/gha-pin.yml

```
@@ -0,0 +1,17 @@
+name: Ruff Linter
```

This really threw me for a loop
```suggestion
name: SHA Pin Check
```